src/compilers/project.json

```diff
@@ -8,8 +8,8 @@
       "type": "platform",
       "version": "1.0.0-rc2-23931"
     },
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160329-01",
-    "Microsoft.Net.Compilers.netcore": "1.3.0-beta1-20160329-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160405-05",
```

Comment:
I'm not sure what this project is for.  I am just doing replacements, not removing.  We can remove them separately if possible.

@brthor @pakrym - do you guys know?